plugin.go

```diff
@@ -150,6 +150,26 @@ func (p *Plugin) build() (err error) {
 	return
 }
 
+func (p *Plugin) test() (err error) {
+	if doesPluginExist(p.filename) && !p.update {
+		return
+	}
+
+	var pass bool
+	if pass, err = goTest(p.gitURL); err != nil {
+		p.out.Error("Test failed :(")
```

Comment:
Might want to pass the error to the client

Reply:
We do, this just prints the status in case we want to continue running on errors (it currently fails on errors)